pip_audit/_service/interface.py

```diff
@@ -54,6 +54,7 @@ class ResolvedDependency(Dependency):
     """
 
     version: Version
+    hash: Optional[str] = None
```

Comment:
This should probably be a `Dict[str, List[str]]` instead, since every requirement is allowed to have multiple hashes, even multiple hashes of the same kind (since `foo==1.2.3` might have both an `sdist` and `bdist_wheel` distribution).

Reply:
Makes sense. I also need to modify the logic to check all of the hashes.